lib/nanoc/base/repos/checksum_store.rb

```diff
@@ -6,7 +6,7 @@ module Nanoc::Int
   class ChecksumStore < ::Nanoc::Int::Store
     # @param [Nanoc::Int::Site] site
     def initialize(site: nil)
-      super('tmp/checksums', 1)
+      super(File.join('tmp', (site.config.env if site).to_s, 'checksums'), 1)
```

Comment:
Can you extract the logic for generating the tmp path into `Nanoc::Int::Store`, as a class method marked `@api private`? Add tests for this extracted method, too. This way, it won’t be duplicated, and it can be tested.